src/App.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import logo from "./logo.png";
+import "./App.css";
+import API from "./WeatherAPI";
+
+//The following is an example of class based implementation of React.
```

Comment:
How was learning class based components?
You could port the project into a vitejs application if you wanted to.